uport-defaults/build.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'java-library'
+apply plugin: "kotlin"
```

Comment:
This project will be a candidate for publishing on jitpack.
For everything to work, it will need some extra plugins
```suggestion
apply plugin: "kotlin"
apply plugin: "maven"
apply plugin: "com.jfrog.bintray"
```